setup.cfg

```diff
@@ -58,7 +58,8 @@ install_requires =
     monai
     tsai; python_version>"3.7"
     importlib-metadata; python_version<"3.8"
-
+    igraph
```

Comment:
Why are these in the default library depends?

Reply:
Sorry I should remove them, since they are in the extra requirements.